src/meta/app/src/principal/user_grant.rs

```diff
@@ -175,8 +179,8 @@ impl GrantEntry {
         &self.object == object
     }
 
-    pub fn has_all_available_privileges(&self) -> bool {
-        let all_available_privileges = self.object.available_privileges();
+    pub fn has_all_available_privileges_without_ownership(&self) -> bool {
```

Comment:
I feel calling `has_all_available_privileges()` is ok, in the current design, having an ownership is not considered as having a privilege.

Reply:
ok. Done